test/src/example.h

```diff
@@ -39,6 +39,7 @@
 
 #include <godot_cpp/classes/control.hpp>
 #include <godot_cpp/classes/global_constants.hpp>
+#include <godot_cpp/classes/sprite2d.hpp>
```

Comment:
This can go in the `.cpp` where it's used.

Reply:
Included your improvements in the new version